### PR TITLE
Shorten README.md in Python data analysis 3rd workout

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Review by nem035 — APPROVED** [review]

File: python-data-analysis/python-data-analysis-core/initializing-and-cleaning-datasets/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Initializing & Cleaning a Dataset
+name: Preparing a Dataset
 
 description: Learn what Series and DataFrames are, and how to initialize and prepare a dataset for analysis.
```
Comment (nem035) [inline python-data-analysis/python-data-analysis-core/initializing-and-cleaning-datasets/README.md:3]: Can we abbreviate the description too?

Would something like this work?

```suggestion
description: Learn how to use Series and DataFrames to prepare a dataset for analysis.
```

Comment (Stefan-Stojanovic) [reply python-data-analysis/python-data-analysis-core/initializing-and-cleaning-datasets/README.md:3]:
```suggestion
description: Learn what Series and Dataframes are, and how to prepare a dataset for analysis.
```
Maybe like this?

Comment (nem035) [reply python-data-analysis/python-data-analysis-core/initializing-and-cleaning-datasets/README.md:3]: Looks good to me

**Review by nem035 — APPROVED** [review]

**Review by nem035 — APPROVED** [review]

**Pull request merged**